requirements/pytorch/test.txt

```diff
@@ -1,18 +1,16 @@
-coverage>=6.4
-codecov>=2.1
-pytest>=7.0
-pytest-cov
-pytest-forked
```

Comment:
@akihironitta Removed the pins a while back because we basically always want to use latest for these dependencies. I would keep it that way.

Reply:
Well, but let's do it in a controlled way as we have now dependable so it sends updated...
otherwise you can say pretty much the same about all our dependencies that we want to use always the latest :rabbit: